README.md

```diff
@@ -1,5 +1,7 @@
 # Chat Location
 
+![app][app_gif]
```

Comment:
thoughts on moving this gif after the description?

Reply:
I took photobooth as an example but I don't mind 😅

Reply:
Actually maybe just move it after the badges?

Reply:
agree